src/pal/src/file/path.cpp

```diff
@@ -1140,7 +1159,7 @@ SearchPathA(
         if (dw == 0 || dw >= MAX_LONGPATH) 
```

Comment:
It seems that since the buffer length is now dynamic, the check dw >= MAX_LONGPATH is obsolete.